rubocop-shopify.gemspec

```
@@ -21,5 +21,5 @@ Gem::Specification.new do |s|
     "allowed_push_host" => "https://rubygems.org",
   }
 
-  s.add_dependency("rubocop", ">= 0.82", "< 0.86")
+  s.add_dependency("rubocop", ">= 0.84", "< 0.86")
```

I think we can enforce 0.85.0 and later already.

```suggestion
  s.add_dependency("rubocop", "~> 0.85.0")
```